### PR TITLE
Dev/homepage completed

### PR DESCRIPTION
**Pull request by felix-melligan**
- Redesigned Header
- Redesigned Footer
- Redesigned Homepage
- Added ContentWrapper for proper Routing around Header and Footer

### CODE REVIEW

File: config/database.yml [hunk 1]
```diff
@@ -15,7 +15,7 @@ default: &default
   pool: <%= ENV.fetch("RAILS_MAX_THREADS") { 5 } %>
   username: postgres
   password:
-  host: ''
+  host: localhost
```
Comment (sbronson) [inline config/database.yml]: seems unrelated

**Comment by felix-melligan** [comment]
Removed the .node_modules folder in my PR (a welcome change)
